neural_style.py

```diff
@@ -10,15 +10,25 @@
 import math
 from argparse import ArgumentParser
 
+from matplotlib import pyplot as plt
```

Comment:
We already have Pillow as a dependency: https://github.com/anishathalye/neural-style#requirements